src/pages/welcome.js

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import {Router} from '@reach/router';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { graphql } from 'gatsby';
+import { httpLink } from '../utils/apollo';
+import gql from 'graphql-tag';
+import {has} from 'lodash';
+
+
+import {Loading} from '../components/organims/Loading';
+import DownloadByOrganization from '../components/organims/DownloadByOrganization';
+import HasBeenDownloaded from '../components/organims/HasBeenDownloaded';
+
+
+const getOrgInfo = gql`
+  query($slug: String!) {
+    organizationBySlug(slug: $slug) {
+      slug,
+      name,
+      domain,
+      pictureUrl,
+    }
+  }
+`;
+
+export default class Welcome extends React.Component{
```

Comment:
I realize it was not documented in the component that we are doing the special technique of templating data coming from Prismic. Worth adding few lines about that for future readers :)

Reply:
Yes did write some minimal info to say some data came from api but no line about the merge of them 👍